drivers/wifi/infineon/airoc_whd_hal_common.c

```diff
@@ -64,21 +64,47 @@ int airoc_wifi_power_on(const struct device *dev)
  * Implement WHD memory wrappers
  */
 
-void *whd_mem_malloc(size_t size)
+inline void *whd_mem_malloc(size_t size)
 {
 	return k_malloc(size);
 }
 
-void *whd_mem_calloc(size_t nitems, size_t size)
+inline void *whd_mem_calloc(size_t nitems, size_t size)
 {
 	return k_calloc(nitems, size);
 }
 
-void whd_mem_free(void *ptr)
+inline void whd_mem_free(void *ptr)
 {
 	k_free(ptr);
 }
 
+inline void whd_mem_memcpy(void *dest, const void *src, size_t len)
+{
+#ifdef PROTO_MSGBUF
```

Comment:
What is the idea of the `PROTO_MSGBUF`? I could not find it being defined anywhere in the code.